packages/harmony/src/storybook/components/ColorPalette/ColorPalette.tsx

```diff
@@ -0,0 +1,79 @@
+import { Text } from 'components/typography/Text'
```

Comment:
small nit for imports in the future, do we want to do `import {Text} from 'components'` ?